sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseUtils.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import org.apache.qpid.proton.message.Message;
+
+/**
+ * This consists of various utilities needed to manage Request/Response  channel.
+ */
+public class RequestResponseUtils {
+    public static final int REQUEST_RESPONSE_UNDEFINED_STATUS_CODE = -1;
```

Comment:
Can this live as a private static method in RequestResponseChannel.

Reply:
Since this is related to reading the status code from AmqpMessages, a better place would be to put it in MessageUtils. There should be a class already

Reply:
RequestResponseUtils.getResponseStatusCode( is used by )  RequestResponseChannel  and also by  ServiceBusMessageSerializer and shared. Thuis I put it in amqp-core and  there is no MessageUtils in amqp-core  .